src/missingarray.jl

```diff
@@ -34,10 +34,10 @@ in(x::Missing, y::CategoricalArray{>:Missing}) = !all(v -> v > 0, y.refs)
 
 function Missings.replace(a::CategoricalArray{S, N, R, V, C}, replacement::V) where {S, N, R, V, C}
     pool = copy(a.pool)
-    v = C(get!(pool, replacement), pool)
+    v = C(pool, get!(pool, replacement))
     Missings.replace(a, v)
 end
 
 function collect(r::Missings.EachReplaceMissing{<:CategoricalArray{S, N, R, C}}) where {S, N, R, C}
```

Comment:
it is a bit confusing that you use `C` to mean what is named `V` above. Maybe make it consistent?

Reply:
Indeed. Better always use the same names as in the type definition.